lib/shouldi/having.ex

```diff
@@ -54,7 +54,11 @@ defmodule ShouldI.Having do
         shouldi_path = unquote(var)[:shouldi_having_path] || []
```

Comment:
Okay, I will use the excellent tests you provided and investigate. Thanks for everything you've done so far!